resolver/src/test/scala/org/globalnames/resolver/model/DataSourcesSpec.scala

```diff
@@ -0,0 +1,49 @@
+package org.globalnames
+package resolver
+package modles
+
+import org.globalnames.resolver.model.DataSources
+import slick.driver.PostgresDriver.api._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class DataSourcesSpec extends SpecConfig {
+  val conn = Database.forConfig("postgresql-test")
+  seed("test_resolver", "ResolverIntegrationSpec")
+
+  override def afterAll(): Unit = {
+    conn.close()
+  }
+
+  describe("DataSources") {
+    val dataSources = TableQuery[DataSources]
+
+    it("contains several records") {
+      whenReady(conn.run(dataSources.size.result)) { res => assert(res > 1) }
+    }
+
+    describe("#id") {
+      it("returns an id of a data source") {
+        whenReady(conn.run(dataSources.take(1).map { _.id }.result)) { res =>
+          res shouldBe Vector(1) }
```

Comment:
I also suggest to wrap trailing `}`. It is easier to see the scope of ending `}` in IDE.